src/reference/asciidoc/kafka.adoc

```diff
@@ -233,7 +233,7 @@ The following example shows how to do so:
 ====
 [source, java]
 ----
-ListenableFuture<SendResult<Integer, String>> future = template.send("something");
+ListenableFuture<SendResult<Integer, String>> future = template.send("myTopic", "something");
```

Comment:
Thanks, but it should be changed to `sendDefault("something")` - we had to rename that method because of a conflict if the key type is `String`.

Reply:
I think what you have is better; merging; thanks for the contribution!!